pack/pack.go

```diff
@@ -151,7 +151,7 @@ func mustCompressBeforeEncryption(file epub.Resource, ep epub.Epub) bool {
 		return true
 	}
 
-	return !strings.HasPrefix(mimetype, "image") && !strings.HasPrefix(mimetype, "video") && !strings.HasPrefix(mimetype, "audio") && !(mimetype == "application/pdf")
+	return !strings.HasPrefix(mimetype, "video") && !strings.HasPrefix(mimetype, "audio") && !(mimetype == "application/pdf")
```

Comment:
What about checking for the `image/png` prefix instead of broadly applying the compression rule to all `image/` files?

Reply:
@danielweck, yes it can be another solution.